substrate/primitives/arithmetic/src/fixed_point.rs

```diff
@@ -513,6 +513,11 @@ macro_rules! implement_fixed {
 				self.0 as f64 / <Self as FixedPointNumber>::DIV as f64
 			}
 
+			/// Convert into a `u32` value.
```

Comment:
Can you add a comment on the rounding mode please? Seems to be `Rounding::Down`.